ansys/dpf/core/check_version.py

```diff
@@ -8,51 +8,48 @@
 import sys
 
 def server_meet_version(required_version, server):
-    """
-    Check if a given server version matches with a required version.
+    """Check if a given server version matches with a required version.
     
     Parameters
     ----------
     required_version : str
-        Required version that will be compared with the server version.
-        
+        Required version to compare with the server version.   
     server : Server
```

Comment:
We could consider a refactor that removes the "name" of the return and just include the `type`.

Reply:
@akaszynski  I think I did this by changing "server : Server" to server : :class:`ansys.dpf.core.server`.